src/components/SearchResultDetail/SearchResultDetail.tsx

```diff
@@ -67,6 +113,15 @@ const SearchResultDetail = ({ id }: { id: string }) => {
           >
             Load Html
           </button>
+          {(!downloadInProgress && (
+            <button
+              data-testid="detail-search-result__download-button"
+              className="button button--primary detail-search-result__button"
+              onClick={downloadPdf}
+            >
+              Download PDF
+            </button>
+          )) || <ClipLoader />}
```

Comment:
This maybe easier to read:

```suggestion

          {downloadInProgress ? <ClipLoader /> : (
            <button
              data-testid="detail-search-result__download-button"
              className="button button--primary detail-search-result__button"
              onClick={downloadPdf}
            >
              Download PDF
            </button>
          )}
```